projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -704,7 +730,10 @@ public function render_custom_email_message_form_field( $type ) {
 		 * @return boolean Indicating if the core invitation main should be sent.
 		 */
 		public function should_send_wp_mail_new_user() {
```

Comment:
Not directly related to this PR: To allow other plugins to manipulate the value of this filter, callbacks should return the value they receive in cases where they don't make changes.

Reply:
@obenland - Something along the lines of (more of a question here) :

```
public function should_send_wp_mail_new_user( $val ) {
			if ( ! isset( $_POST['invite_user_wpcom'] ) && isset( $_POST['send_user_notification'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Missing
				return true;
			}
			return $val;
		}
```

Thanks for the tip. Would that still be applicable here or is this a tip to keep in mind for other places where it may make more sense to fallback to the value the callback received?

Reply:
Yeah, this one is interesting. [`true` is the default value](https://github.com/WordPress/wordpress-develop/blob/6.4/src/wp-includes/pluggable.php#L2210-L2210), so I guess it would be the return within the if clause that should return `$send`? 

You're disabling the new user notification for all cases unless you allow it to by setting the `send_user_notification` query arg?

> Would that still be applicable here or is this a tip to keep in mind for other places

Applicable for this callback, I just wanted to acknowledge that it's not something that was caused by this PR.

Reply:
Yes, I'm disabling the new user email notification only if the `send_user_notification` query arg does not exist. This makes more sense now. Thanks.